1 tables.sql

```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(
+	GermName varchar(200) not null,
+	Host varchar(20)

```

Comment:
this should not allow `null`, also we need a check constraint as is in the spec

Reply:
added the not null

Reply:
this is done, I am resolving the conversation